plugins/modules/packaging/os/zypper.py

```diff
@@ -164,7 +178,14 @@
   zypper:
     name: '*'
     state: dist-upgrade
-    extra_args: '--no-allow-vendor-change --allow-arch-change'
+    allow_vendor_change: true
+    extra_args: '--allow-arch-change'
+
+- name: Perform a installaion of nmap with the install option replacefiles and allow_vendor_change
```

Comment:
nit: you should remove allow_vendor_change in the name of the task.

Reply:
Done in 78326ce